custom_components/truenas/__init__.py

```diff
@@ -227,6 +234,33 @@ def device_info(self):
         }
 
 
+class TrueNASPoolEntity:
+    """Represents a pool on the TrueNAS host."""
+
+    _pool: CachingPool
+
+    @property
+    def device_info(self):
+        return {
+            "identifiers": {
+                (DOMAIN, self._pool.guid),
+            },
+            "name": f"TrueNAS Pool - {self._pool.name}",
+        }
+
+    @property
+    def extra_state_attributes(self):
+        """Return extra Pool attributes"""
+        encryption = bool(self._pool.encrypt)
+        is_decrypted = bool(self._pool.is_decrypted)
+        return {
+            ATTR_POOL_NAME: f"{self._pool.name}",
+            ATTR_POOL_ID: f"{self._pool.id}",
```

Comment:
I don't think the pool ID is very helpful.  Is there a reason you are including it here?